pkg/processor/metricfrequencyprocessor/pdata.go

```diff
@@ -0,0 +1,24 @@
+package metricfrequencyprocessor
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"sort"
+)
+
+type PdataTimestampArr []pdata.Timestamp
+
+func (pta PdataTimestampArr) Len() int {
```

Comment:
BTW, why do we need to export it?

Reply:
it's part of the sort `Interface` and reducing visibility is not an option.

I think I can make the helper type not exported, though.